src/Network/Network.Test/Network.Test.csproj

```diff
@@ -14,11 +14,11 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.3.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="201.359.57612" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="3.1.0-preview.2" />
     <PackageReference Include="Microsoft.Azure.Insights" Version="0.16.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="39.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="43.0.0-preview" />
```

Comment:
this might potentially have impact on other teams working on this branch

Reply:
The Swagger was checked-in in the network-november-release. Until the official SDK is generated, the tests don't pass, right? I plan on reverting those files once that has been done if that's alright. Or I can revert it now?

Reply:
What would you recommend? @VeryEarly

Reply:
> What would you recommend? @VeryEarly
the nework SDK is fine, what I mean is the compute SDK.

It looks like a major version upgrade and it's possible that other test cases are not compatible with it.

Reply:
Anyway let's run the CI and see.